packages/zudoku/src/lib/plugins/openapi/OperationList.tsx

```diff
@@ -154,7 +155,15 @@ export const OperationList = () => {
       {result.data.schema.tags
         .filter((tag) => tag.operations.length > 0)
         .map((tag) => (
-          <div key={tag.name}>
+          <div
+            key={tag.name}
+            style={
+              {
+                "content-visibility": "auto",
+              } as unknown as CSSProperties
+              // we need this until we have types
+            }
+          >
```

Comment:
We could also use Tailwind's [arbitrary properties](https://tailwindcss.com/docs/adding-custom-styles#arbitrary-properties).


```suggestion
          <div key={tag.name} className="[content-visibility:auto]">
```